### PR TITLE
Update module name to agree with new repo location

### PR DESCRIPTION
**Pull request by carbonin**

### CODE REVIEW

**Comment by carbonin** [comment]
/lgtm

Doing this myself until we get more of the basic changes done.

**Comment by carbonin** [comment]
This should probably have been considered part of https://issues.redhat.com/browse/MGMT-15326

File: internal/imageserver/imageserver.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// if anything fails remove the workdir, if create succeeds it will remove the workdir so this will be a noop
 	defer os.RemoveAll(isoWorkDir)
 
-	// TODO: carbonin improve this to wait for some timout (use ctx?) instead of erroring on a lock failure immediately
+	// TODO: openshift improve this to wait for some timout (use ctx?) instead of erroring on a lock failure immediately
```
Comment (osherdp) [inline internal/imageserver/imageserver.go:57]: nit: I guess this replace isn't on purpose

**Review by osherdp — APPROVED** [review]

File: internal/imageserver/imageserver.go (hunk 1 above)
Comment (carbonin) [reply internal/imageserver/imageserver.go:57]: :man_facepalming:

**Pull request merged**